tests/Fixtures/TestBundle/GraphQl/Type/Definition/DateTimeType.php

```diff
@@ -79,7 +79,7 @@ public function parseValue($value)
      */
     public function parseLiteral($valueNode, ?array $variables = null)
     {
-        if ($valueNode instanceof StringValueNode) {
+        if ($valueNode instanceof StringValueNode && false !== \DateTime::createFromFormat(\DateTime::ATOM, $valueNode->value)) {
```

Comment:
shouldn't this format be configurable? Or mb a documentation to override this?

Reply:
It‘s just an example (inside the `tests` folder).